### PR TITLE
Add new alpine images for OpenSSL 1.1 and 3.0

### PR DESCRIPTION
**Pull request by pimeys**

### CODE REVIEW

**Review by pimeys — COMMENTED** [review]
Could we also just not build the debian images in this PR? Just comment them out in the buildkite pipeline.

File: musl/libssl1.1.x.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+RUN wget -c https://www.openssl.org/source/openssl-1.1.1s.tar.gz
+RUN tar -xzvf openssl-1.1.1s.tar.gz
+
+RUN cd openssl-1.1.1s && ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl shared zlib && make -j8
+
+RUN cd openssl-1.1.1s && make install
+RUN cd /etc && echo "/usr/local/ssl/lib:/lib:/usr/lib:/usr/local/lib" > ld-musl-x86_64.path
+
+ENV OPENSSL_DIR /usr/local/ssl
+ENV OPENSSL_LIB_DIR /usr/local/ssl/lib
```
Comment (janpio) [inline musl/libssl1.1.x.Dockerfile:19]: Just to make this explicit:
Why are we switching to building OpenSSL ourselves?

Comment (pimeys) [reply musl/libssl1.1.x.Dockerfile:19]: We've been doing that with Debian and RHEL already, see:

https://github.com/prisma/engine-images/blob/main/rhel/libssl1.1.x.Dockerfile

Now we have full control which version we use, and we can use an old(er) Alpine base with whatever SSL we desire. This way we get a wider support of Alpines with a specific OpenSSL version.

**Comment by pimeys** [comment]
can you approve @aqrln?

**Review by aqrln — APPROVED** [review]

**Pull request merged**